suite-native/graph/src/components/Graph.tsx

```diff
@@ -138,7 +143,11 @@ export const Graph = <TGraphPoint extends FiatGraphPoint>({
             />
             {loading && (
                 <Box style={applyStyle(graphMessageStyleContainer)}>
-                    <Loader title="Loading graph data..." />
+                    <Loader
+                        title={translate(
+                            `graph.loader.${isDiscoveryActive ? 'discovery' : 'loadingData'}`,
```

Comment:
Please never use dynamic generated keys, it's impossible to detect unused keys by static analysis and other kind of issues. Use some kind Map instead `{state1: "transalationId1", state2: "translationId2" }`

Reply:
After team slack discussion, there is no need for special loading message for discovery. So the dynamic selection was removed in [da8a59d](https://github.com/trezor/trezor-suite/pull/10030/commits/da8a59d889d710e58aee11beab14f0dcc0c0cdf1)